build/steps/build-windows.yml

```diff
@@ -52,7 +56,7 @@ jobs:
         solution: ${{ parameters.slnPath }}
         platform: '$(BuildPlatform)'
         configuration: '$(BuildConfiguration)'
-        msbuildArguments: ${{ parameters.msbuildExtraArguments }}
+        msbuildArguments: ${{ parameters.msbuildExtraArguments }} /p:JavaSdkDirectory="$(JAVA_HOME_8_X64)"
```

Comment:
https://developercommunity.visualstudio.com/content/problem/606218/error-xa5300-the-java-sdk-directory-could-not-be-f.html